tests/SCF/test_SCF.py

```diff
@@ -1,19 +1,89 @@
-import pytest
+from pytest import approx
 import SCF
+import pytest
+from pyscf import gto  # PySCF is a quantum chemistry python module
+import numpy as np
+import sys
+sys.path.append("/Users/mkx/Desktop/Courses/Python/HF_SCF_Assignment")
+print(sys.path)
+mol_h2o = gto.M(unit="Bohr",

```

Comment:
should not do this, this should use the fixture as was shown in class and through the example function. This is already defined in `conftest.py`